### ISSUE TITLE
Can not tag -- Error: "Can not set property 'version' of undefined"

### ISSUE THREAD

**Issue by jhamlet** [teambit/bit#1364]
Trying to tag a component after updating docs.

## Status
```
$ bit status
modified components
(use "bit tag --all [version]" to lock a version with all your changes)
(use "bit diff" to compare changes)

     > util/dispatch ... ok


components pending to be tagged automatically (when their dependencies are tagged)
     > jhamlet.ink/util/store ... ok
```

## Result of Trying to Tag

```
$ bit tag -v -m "Updating docs" util/dispatch
⢂⠀ building components(node:66342) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
Cannot set property 'version' of undefined
```

I've tried `--all`, tagging `util/store` first, and using the `-v` verbose flag to get more useful information but nothing more than the above is shown.

## Steps to Reproduce the Problem

  1. [Repository branch](https://github.com/jhamlet/ink/tree/cannot-bit-tag)
  1. Commands used are above

## Specifications

  - Bit version: 13.0.5
  - Node version: v10.9.0
  - npm / yarn version: 6.2.0
  - Platform: Mac OS X 10.13.6 (17G4015)
  - Bit compiler (include version): bit.envs/compilers/babel@0.0.20
  - Bit tester (include version): bit.envs/testers/jest@0.0.35

**Comment by GiladShoham** [comment]
I was able to reproduce the issue with bit@13.0.5, however, when trying with master it worked as expected.
(@jhamlet do not try to use master since this will affect your entire workspace, and you won't be able to export it to bitsrc until we release the version).
The issue has been resolved, but I'll try to see if I can find a workaround for you until we release the version.

**Comment by GiladShoham** [comment]
As a quick workaround, you can import util/dispatch into a temp folder, make the change, tag and export it. Then import the new version on your repo.
This is not a great solution, I know.
But as I said before it's already solved for the next version.
I tested this myself and it worked.

**Comment by davidfirst** [comment]
This has been fixed and released a while ago (v14.0.0 I believe).

**Comment by GiladShoham** [comment]
@jhamlet If you still face this with the new version (14.0.1), please re-open.
